File: KnowledgeCenterClient/src/app/why-stay-home/models/CovidStats.ts
export interface CovidStatsResponse {
  lastDay: Date;
  france: GlobalStats;
  world: GlobalStats;
  deaths: CovidStats;
  recovered: CovidStats;
  detected: CovidStats;
}

export interface GlobalStats {
  totalDeaths: number;
  dailyNewDeaths: number;
  dailyDeathsVariation: number;

  totalRecovered: number;
  dailyNewRecovered: number;
  dailyRecoveredVariation: number;

  totalDetected: number;
  dailyNewDetected: number;
  dailyDetectedVariation: number;
}

export interface CovidStats {
  chartDataTotal: ChartData;
  chartDataPerDay: ChartData;
}

export interface ChartData {
  labels: Array<string>;
  dataSets: Array<DataSet>;
  total: number;
}

export interface DataSet {
  label: string;
  data: Array<number>;
}

File: KnowledgeCenterServer/_Covid/KnowledgeCenter.Covid.Contracts/CovidStats.cs
using System;
using System.Collections.Generic;

namespace KnowledgeCenter.Covid.Contracts
{
    public class CovidStatsResponse
    {
        public DateTime LastDay { get; set; }

        public GlobalStats France { get; set; }
        public GlobalStats World { get; set; }

        public CovidStats Deaths { get; set; }
        public CovidStats Recovered { get; set; }
        public CovidStats Detected { get; set; }
    }

    public class GlobalStats
    {
        public int TotalDeaths { get; set; }
        public int DailyNewDeaths { get; set; }
        public decimal DailyDeathsVariation { get; set; }

        public int TotalRecovered { get; set; }
        public int DailyNewRecovered { get; set; }
        public decimal DailyRecoveredVariation { get; set; }

        public int TotalDetected { get; set; }
        public int DailyNewDetected { get; set; }
        public decimal DailyDetectedVariation { get; set; }
    }

    public class CovidStats
    {
        public ChartData ChartDataTotal { get; set; }
        public ChartData ChartDataPerDay { get; set; }
    }

    public class ChartData
    {
        public List<string> Labels { get; set; }
        public List<DataSet> DataSets { get; set; }
    }

    public class DataSet
    {
        public string Label { get; set; }
        public List<int> Data { get; set; }
    }
}
